File: 프로그래머스/Lv.1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    for l in commands :
        i = l[0]
        j = l[1]
        k = l[2]
        
        print(array[i-1:j])
        
        temp = array[i-1:j]
        temp.sort()
        temp = temp[k - 1]
        
        answer.append(temp)
    return answer

File: 프로그래머스/1/42748. K번째수/K번째수.java
import java.util.*;

class Solution {
    public static int[] solution(int[] array, int[][] command) {
        List<Integer> list = new ArrayList<>();
        for (int[] ints : command) {
            int i = ints[0];
            int j = ints[1];
            int k = ints[2];

            int[] ints1 = Arrays.copyOfRange(array, i - 1, j);
            Arrays.sort(ints1);
            list.add(ints1[k - 1]);
        }

        return list.stream()
                .mapToInt(i -> i).toArray();
    }
}